README.md

```diff
@@ -11,189 +12,67 @@ This project explores the use of large language models (LLMs) for product recomm
 - **Sequential Recommendation System**: Predicts the next (maximum) 10 items a user might purchase in the future, based on their review history.
 - **Custom Data Processing**: Handles temporal data (order by time to prevent data/target/information leakage) and processes the sequence of user interactions, ensuring proper padding and handling of missing data.
 - **Fine-tuning GPT-2**: Fine-tuned GPT-2 medium model to handle recommendation tasks using user reviews, product metadata, and timestamps.
-- **Evaluation Metrics**: Supports evaluation with metrics such as Recall, Precision, MRR and more.
+- **Evaluation Metrics**: Supports evaluation with metrics such as Recall, Precision, MRR, HR@10, and NDCG@10.
 - **Custom Padding Strategies**: The model can handle missing data with customizable padding strategies (repeat, special token, or no padding).
 
 #### Achieved Metrics:
-- **Precision@10**: $2.2$%
-- **Recall@10**: $5.5$%
-- **Mean Reciprocal Rank (MRR)**: $0.04$
+- **Precision@10**: 2.19%
```

Comment:
Please round the metrics to 1 decimal place for easy interpretation (e.g. 2.2% instead of 2.19%)